MH4U Dex/Base.lproj/Main.storyboard

```diff
@@ -156,6 +156,15 @@
                                     <segue destination="hfi-eK-gsg" kind="show" id="dX6-Hf-8Bu"/>
                                 </connections>
                             </button>
+                            <button opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Cxa-Tg-f1x">
+                                <rect key="frame" x="35" y="377" width="136" height="30"/>
+                                <state key="normal" title="RESET CORE DATA">
```

Comment:
Thanks.  I've put a TODO: for that in the view controller's implementation file.  I might move or change the buttons so I won't hook them up to IBOutlets just yet.